docs/source/raft_ann_benchmarks.md

```diff
@@ -0,0 +1,169 @@
+
+mamba install -c rapidsai libraft-ann-bench
+```
+The channel `rapidsai` can easily be substituted `rapidsai-nightly` if nightly benchmarks are desired.
+
+Please see the [build instructions](ann_benchmarks_build.md) to build the benchmarks from source.
+
+## Running the benchmarks
+
+### Usage
+There are 4 general steps to running the benchmarks:
+1. Prepare Dataset
+2. Build Index and Search Index
+3. Evaluate Results
+4. Plot Results
+
+### Python-based Scripts
+We provide a collection of lightweight Python based scripts that are wrappers over
+lower level scripts and executables to run our benchmarks. Either Python scripts or
+[low-level scripts and executables](ann_benchmarks_low_level.md) are valid methods to run benchmarks,
+however plots are only provided through our Python scripts.
+#### End-to-end example: Million-scale
+```bash
+# All scripts are present in directory raft/scripts/ann-benchmarks
+
+# (1) prepare dataset
+python scripts/ann-benchmarks/get_dataset.py --name glove-100-angular --normalize
+
+# (2) build and search index
+python scripts/ann-benchmarks/run.py --configuration conf/glove-100-inner.json
+
+# (3) evaluate results
+python scripts/ann-benchmarks/data_export.py --output out.csv --groundtruth data/glove-100-inner/groundtruth.neighbors.ibin result/glove-100-inner/
+
+# (4) plot results
+python scripts/ann-benchmarks/plot.py --result_csv out.csv
+```
+
+#### End-to-end example: Billion-scale
+The above example does not work at Billion-scale because [data preparation](#prep-dataset) is not yet
+supported by `scripts/get_dataset.py`. To download and prepare [billion-scale datasets](ann_benchmarks_low_level.html#billion-scale),
+please follow linked section. All other python scripts mentioned below work as intended once the
+billion-scale dataset has been downloaded.
+To download Billion-scale datasets, visit [big-ann-benchmarks](http://big-ann-benchmarks.com/neurips21.html)
+
+```bash
+mkdir -p data/deep-1B && cd data/deep-1B
+# (1) prepare dataset
+# download manually "Ground Truth" file of "Yandex DEEP"
+# suppose the file name is deep_new_groundtruth.public.10K.bin
+../../scripts/split_groundtruth.pl deep_new_groundtruth.public.10K.bin groundtruth
+# two files 'groundtruth.neighbors.ibin' and 'groundtruth.distances.fbin' should be produced
+
+# (2) build and search index
+python scripts/run.py --configuration conf/deep-1B.json
+
+# (3) evaluate results
+python scripts/data_export.py --output out.csv --groundtruth data/deep-1B/groundtruth.neighbors.ibin result/deep-1B/
+
+# (4) plot results
+python scripts/plot.py --result_csv out.csv
+```
+
+##### Step 1: Prepare Dataset<a id='prep-dataset'></a>
+The script `scripts/ann-benchmarks/get_dataset.py` will download and unpack the dataset in directory
+that the user provides. As of now, only million-scale datasets are supported by this
+script. For more information on [datasets and formats](ann_benchmarks_low_level.html#bash-prepare-dataset).
```

Comment:
I think the new "billino-scale datasets guide" should also include the million-scale instructions as wel. Just create an 'ann_bench_datasets.md" or something.